### PR TITLE
Display upload progress

### PR DESCRIPTION
**Pull request by LinnierGames**
### What's new
- Create Navigator to setup and present the Add Photo flow
- Create AddPhotoUploadSession Vc

### What has changed
- Present the AddPhoto Vc in a Navigation Vc

### CODE REVIEW

**Review by LinnierGames — COMMENTED** [review]
Self-reviewed

**Comment by LinnierGames** [comment]
[Display a spinner while images are sending](https://trello.com/c/hf5wS1na/69-display-a-spinner-while-images-are-sending)

File: PicPi/AddPhotoUploadSessionViewController.swift [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  }
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    view.backgroundColor = .white
+    title = "Add Photos"
+    navigationItem.hidesBackButton = true
+    setupProgressLabel()
+    setupDoneButton()
```
Comment (RawiSoft) [inline PicPi/AddPhotoUploadSessionViewController.swift]: It's a little bit confusing
SetupDoneButton , the button is searchButton and the button title is Cancel , if you don't mind fix that :)

**Review by RawiSoft — CHANGES_REQUESTED** [review]

File: PicPi/AddPhotoUploadSessionViewController.swift (hunk 1 above)
Comment (LinnierGames) [reply PicPi/AddPhotoUploadSessionViewController.swift]: Oops, that's wha happens when I copy and paste code, fixed

**Pull request merged**